### PR TITLE
feat: delete member modal

### LINKED ISSUE #8: [FEAT] Organization settings  Manage Members > Implement this Delete Member modal
## Description
Implement the "Delete Member" modal on the member page.

## Acceptance Criteria
- The "Delete Member" modal should have:
  - A heading "Delete Member" properly aligned
  - A message: "Are you sure you want to delete Chad Bosewick? All of your data will be permanently removed. This action cannot be undone."
  - Two buttons at the bottom: "Cancel" and "Delete"
- Clicking outside the modal and clicking the times button should close the modal
- Proper usage of overlay opacity on the parent page when the modal is open
- The modal adheres to the brand colors and design
- The modal is responsive across device widths

## Expected Outcome
- User should see a properly aligned "Delete Member" modal with the specified text and buttons
- Clicking "Cancel" should close the modal
- Clicking "Delete" should delete the member and close the modal
- Clicking outside the modal or the times button should close the modal
- The modal should be responsive

## Purpose
As an administrator, I want to delete members from the workspace with a clear confirmation process.

## Requirement
- Implement the "Delete Member" modal with the specified text and buttons
- Ensure the modal closes when clicking outside it or the times button
- Ensure overlay opacity is applied to the parent page when the modal is open
- Ensure the modal follows the brand's design guidelines
- Make sure the modal is responsive

## Task
- [x] Create "Delete Member" modal with specified heading, message, and buttons
- [x] Implement functionality to close the modal when clicking outside it or the times button
- [x] Ensure overlay opacity is applied to the parent page when the modal is open
- [x] Ensure the modal follows brand colors and design
- [x] Ensure the modal is responsive

## Picture
Confirm Delete

## Dev Mode
- [Dev Mode Link](https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=96-100990&m=dev)

### PR DESCRIPTION
**Pull request by monyasau**
# Description
Delete member modal, I added the component to index file(i have removed it from the route file), because the settings page is not ready as at the time I'm making this pr

**Closes #8**

# Changes proposed

## What were you told to do?
Develop reusable delete member modal

## What did you do?
Delete member modal

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos
Screenshot 2024-07-20 214810
Screenshot 2024-07-21 092806

Video: https://github.com/user-attachments/assets/54c6ea23-a9a4-4c7f-893a-2d358a6576e7

### CODE REVIEW

File: app/routes/_index.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { useState } from "react";
 
 import { Button } from "~/components/ui/button";
 import CardPlatform from "~/components/ui/card/card-platform";
+import DeleteMemberModal from "~/components/ui/modals/DeleteMemberModal";
 import OtpAuth from "~/components/ui/otp/OtpAuth";
 import { Input } from "~/types/otpauth";
```
Comment (chumex412) [inline app/routes/_index.tsx:9]: Please look for the person using your component on their page, and tell them to integrate it there. This route isn't where you do that.

**Review by chumex412 — CHANGES_REQUESTED** [review]

File: app/routes/_index.tsx (hunk 1 above)
Comment (monyasau) [reply app/routes/_index.tsx]: @chumex412 i have fixed the issues you talked about

**Comment by SirhmVFX** [comment]
@chumex412

**Comment by monyasau** [comment]
> @chumex412

@chumex412 could you do the review

**Pull request merged**